packages/router/src/config.ts

```diff
@@ -246,14 +246,14 @@ export const getEnvConfig = (crossChainData: Map<string, any> | undefined): Nxtp
       nxtpConfig.chainConfig[chainId].priceOracleAddress = res?.address;
     }
 
-    if (!chainConfig.minGas) {
+    if (chainConfig.minGas === undefined) {
```

Comment:
won't work for `null` -- should do `typeof` probably

Reply:
It won't be null though, it's parsed from JSON or env vars.

Reply:
technically you can pass null in to JSON so safest thing is `chainConfig.minGas === undefined || chainConfig.minGas === null`

Reply:
I think we don't need to change first two conditions. because their types are string and number, not boolean.
- if (!chainConfig.minGas) 
- if (!chainConfig.relayerFeeThreshold)

But we have to absolutely revise allowFulfillRelay part. because type of allowFulFillRelay variable is boolean. so it should be `if (!chainConfig.allowRelayFulfill === undefined)`

Reply:
if (!chainConfig.minGas)

Reply:
> I think we don't need to change first two conditions. because their types are string and number, not boolean.

Correct, I removed the change on the first two.

> so it should be if (!chainConfig.allowRelayFulfill === undefined)

It would need to be `chainConfig.allowFulfillRelay === undefined` because you only want to set the default if it's undefined. We don't need that extra !.

Reply:
Correct. `!` was my mistake.